themes/spaceship.omp.json

```diff
@@ -16,9 +16,9 @@
           "invert_powerline": false,
           "foreground": "lightYellow",
           "properties": {
-            "display_host": false,
+            "display_host": true,
```

Comment:
this property has become obsolete, please use a `template` ;-)

Reply:
Thank you! Can you check again pls?

Reply:
> 

Thank you! Can you check again pls?